### ISSUE TITLE
New term request, 'has disease driver'

### ISSUE THREAD

**Issue by lschriml** [oborel/obo-relations#506]
I am requesting a new relation term to define the relationship between a disease driver and a disease. 
For the Human Disease Ontology project. 

Definition: A relation between a material entity and a disease of a host, in which the material entity is not part of the host itself, and the condition results in pathological processes. 

Aligning this definition to 'has allergic trigger'.  

Thank you,
Lynn Schriml

**Comment by nlharris** [comment]
Is this still needed? Who can address this?

**Comment by matentzn** [comment]
This issue is only a few days old.. @lschriml can address it when she starts working on patternisation of her driver classes, and we can help.

**Comment by lschriml** [comment]
@nlharris and @matentzn -- Yes, the term is still needed,  What do you need for this RO term to be created ? 

Cheers,
Lynn

**Comment by matentzn** [comment]
@lschriml i think the main thing is to decide whether we can omit the `disease` and simply use `has driver`. This will make it easier to integrate with other modelling patterns.

Also remember that `has driver` connects the disease with the driver - not the driver with the element (like alcohol).

My preferred option: you provide all the metadata here:

- label
- definition
- orcid
- comment with intended use case
- rdfs:seeAlso to this github issue

Then we make this the first test for our new template based pipeline, which will make @cthoyt happy.

**Comment by lschriml** [comment]
Hello @matentzn: 
      I see your point. 
 
 label:   has_driver: 
 Definition: A relation between a material entity and a disease of a host, in which the material entity is not part of the host itself, and the condition results in pathological processes.
 
My ORCID: 0000-0001-8910-9851

rdfs:seeAlso https://github.com/oborel/obo-relations/issues/506

**Comment by matentzn** [comment]
Werent we going for some model like that?

`fetal alcohol disorder`--[`has driver`]-->`alcohol driver`--[`has stressor?`]-->`alcohol`

**Comment by lschriml** [comment]
yes, I think that is the modeling you proposed and we discussed.
  I see how it is built so that the ontology connections make sense. 

However, from an epidemiological perspective, the alcohol is the driver, I am not sure how we would define 'stressor' 
     -- as the 'driver' is the thing that is causing the pathophysiology end result 
 
What I really want to model is that a driver (e.g. alcohol) is 'disease driver' resulting in a disease (e.g. fetal alcohol syndrome).

**Comment by lschriml** [comment]
Thinking of modeling consistency: 
    I was modeling the requested RO term name based on the RO term, 'has allergic trigger', 
              where the modeling is: disease (e.g. asthma) - 'has allergic trigger' some pollen
thus
        for disease drivers: 
                                             disease -- 'has disease driver' some alcohol

**Comment by matentzn** [comment]
In this case, we need something like this:

You curate:
```
`fetal alcohol disorder`--[`realised in response to`]-->`alcohol driver`--[`has stimulus`]-->`alcohol` 
```

You can use has stimulus to be consistent with ECTO, and the`driver` class takes care of the driver semantics (i.e. the fact that it is not _just_ a stimulus).

You infer:

```
`fetal alcohol disorder`--[`has driver`]-->`alcohol` 
```

Using these axioms:
```
driver SubClassOf isDriver some self
`has driver` subChainOf `realised in response to` o `isDriver` o `has stimulus`
```

This is a bit sophisticated, @balhoff what do you think of this?

**Comment by lschriml** [comment]
Getting back to this request.

I would like to have 'has disease driver' added to the RO. 
 
Definition: A relation between a material entity and a disease of a host, in which the material entity is not part of the host itself, and the condition results in pathological processes.

Aligning this definition to 'has allergic trigger' RO term. 

Cheers,
Lynn

**Comment by matentzn** [comment]
You can add the term using Protege in the appropriate place!

**Comment by lschriml** [comment]
Hello @matentzn -- I'm not sure I understand your message. 
I am not a RO developer. 
Can one of the RO developers please add this new term. 

Thank you,
Lynn

**Comment by matentzn** [comment]
No worries. RO is a community-edited ontology, I don't think there is an RO developer per se - we all have to do our own additions and pull requests usually.

**Comment by lschriml** [comment]
Ok. Thanks Nico.
So should I go ahead and make a pull request on RO and add the RO term ?

Do you want me to add a more generic ‘has driver’ as well ? 

Cheers,
Lynn

**Comment by matentzn** [comment]
Yes, maybe add both. Also, you first need to add yourself yo this file:

https://github.com/oborel/obo-relations/blob/master/src/ontology/ro-idranges.owl

To secure an ID range. Make a pull request and you can tag me as a reviewer and I will take care of the rest

**Comment by lschriml** [comment]
@matentzn -- I have added the new terms, and initiated a pull request

**Comment by lschriml** [comment]
@cmungall @matentzn -- Looking forward to having these two new RO terms added to the next RO release. 
    -- Requesting the final review of #531 pull request, so that these terms can be used for annotations.

Cheers,
Lynn

**Comment by matentzn** [comment]
I am on leave now for quite a while - keep pinging @cmungall here to get it merged. Thanks!

**Comment by lschriml** [comment]
Thank you @matentzn -- Happy Holidays !!

@cmungall  -- Chris -- please let me know if I can provide any further info to get these RO terms added.

Cheers,
Lynn

**Comment by nlharris** [comment]
Closed by #531
